lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -492,16 +492,18 @@ class MailboxController extends BaseMailboxController with MailboxActionHandlerM
     _openMailboxEventController.add(OpenMailboxViewEvent(context, presentationMailboxSelected));
   }
 
-  void goToCreateNewMailboxView(BuildContext context) async {
+  void goToCreateNewMailboxView(BuildContext context, PresentationMailbox? parentMailbox) async {
```

Comment:
I'm not a fan of this way of writing, it's confusing when others read it.
```suggestion
void goToCreateNewMailboxView(BuildContext context, {PresentationMailbox? parentMailbox})
```